src/uwtools/api/template.py

```diff
@@ -1,14 +1,14 @@
-from typing import Dict, Optional
+from typing import Dict, Optional, Union
 
 from uwtools.config.jinja2 import render as _render
 from uwtools.types import DefinitePath, OptionalPath
 
 
 def render(
-    input_file: OptionalPath,
-    output_file: OptionalPath,
-    values_file: DefinitePath,
+    values: Union[dict, DefinitePath],
```

Comment:
The API analog to `uw template render` can now accept an in-memory `dict` as a source for values to render the template, along with the existing file-based values option.